cips/versioned_gas_scheduler_variables.md

```diff
@@ -0,0 +1,56 @@
+| cip | 24 |
+| - | - |
+| title | Versioned Gas Scheduler Variables |
+| description | Transition to hard fork-only updates for gas scheduler variables |
+| author | Nina Barbakadze (@ninabarbakadze) |
+| discussions-to |  |
```

Comment:
Needs `discussions-to: The url pointing to the official discussion thread` per CIP-1: https://cips.celestia.org/cip-1.html#discussions-to-header